libraries/Microsoft.Bot.Builder.Dialogs/DialogEvents.cs

```diff
@@ -10,5 +10,8 @@ public class DialogEvents
         public const string CancelDialog = "cancelDialog";
         public const string ActivityReceived = "activityReceived";
         public const string Error = "error";
+        public const string BeginTemplateEvaluation = "beginTemplateEvaluation";
```

Comment:
is it possible to move those two events into maybe LGEvents?

Reply:
Yes this should NOT be in this assembly

---
In reply to: [416500764](https://github.com/microsoft/botbuilder-dotnet/pull/3791#discussion_r416500764) [](ancestors = 416500764)

Reply:
This is a REQUIRED change, you cannot put this here.

---
In reply to: [417385145](https://github.com/microsoft/botbuilder-dotnet/pull/3791#discussion_r417385145) [](ancestors = 417385145,416500764)